### PR TITLE
Add NixOS Layer

### PR DESCRIPTION
**Pull request by CestDiego**
Hey NixOS users! @travisbhartwell and @cestdiego (me) have been developing nixos packages to let Emacs be the ultimate tool in NixOS :P Hope you can give feedback... The layer doesn't work but will once the packages get accepted into the MELPA repo.

### CODE REVIEW

**Comment by jb55** [comment]
awesome!

**Comment by robbyoconnor** [comment]
<3

File: contrib/nixos/README.org [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+     - [[#layer][Layer]]
+ - [[#key-bindings][Key Bindings]]
+     - [[#nixos-options][NixOS Options]]
+
+* Description
+
+This layer adds tools for better integration of emacs in NixOS.
+
+** Features:
+- Nix-mode using  [[https://github.com/travis/anaconda-mode][nix-mode]]
```
Comment (trishume) [inline contrib/nixos/README.org]: This link is weird and broken

**Comment by trishume** [comment]
:+1:

File: contrib/nixos/README.org (hunk 1 above)
Comment (CestDiego) [inline contrib/nixos/README.org]: Thx, Correcting now..

**Comment by syl20bnr** [comment]
Good job @travisbhartwell and @CestDiego ! :+1: 
Thank you !
Cherry-picked into `develop` branch, you can safely delete your branch.

**Pull request closed without merging**